README.md

```diff
@@ -122,17 +122,17 @@ yum install mosquitto
 
 See [mosquitto official website](https://mosquitto.org/download/) for more information.
 
-### Build
+### Build Edge
 
 Clone kube-edge
```

Comment:
Please make the names consistent (low prio) : kubeedge / KubeEdge

Reply:
Fixed